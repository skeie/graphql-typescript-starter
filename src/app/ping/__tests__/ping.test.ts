import createServer from '../../../../test/support/createServer';
import gql from 'graphql-tag';
import { db, initDb, cleanDb } from '../../../../test/support/dbHelpers';

const PING_QUERY = gql`
  query Ping {
    ping
  }
`;

describe('Ping', () => {
  beforeAll(initDb);
  afterAll(cleanDb);

  it('Ping ping', async () => {
    const { query } = await createServer();
    const res = await query({ query: PING_QUERY });
    expect(res).toMatchSnapshot();
  });
});
