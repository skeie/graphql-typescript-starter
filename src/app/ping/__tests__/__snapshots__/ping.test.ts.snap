// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ping Ping ping 1`] = `
Object {
  "data": Object {
    "ping": "pong",
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
